src/index.js

```diff
@@ -56,8 +56,12 @@ export default function auth(config = {}) {
       config.local = {};
     }
 
+    if (config.cookie ){
+      config.cookie = Object.assign({}, defaults.cookie, config.cookie);
+    }
+    
     // Merge and flatten options
-    const authOptions = Object.assign({}, defaults, app.get('auth'), config);
```

Comment:
I think we do need this back. The reason being is you can just do `app.configure(authentication())` if you want to use all the default options **plus** your auth config defined in `default.json` or `production.json`. For example, your token secret, etc.